app/viewmodels/ResultsPageSummary.scala

```diff
@@ -25,54 +25,72 @@ import utils.Utils.{emptyString, period}
 
 import java.time.LocalDate
 
-class ResultsPageSummary(from: LocalDate, to: LocalDate, day: Boolean = true)
+class ResultsPageSummary(from: LocalDate, to: LocalDate, isDay: Boolean = true)
                         (implicit messages: Messages) extends SummaryListRowHelper {
 
-  def rows(fullStop: Boolean = true): SummaryListRow = {
-    cashTransactionsResultRow(CashTransactionDates(from, to), fullStop, day)
+  def rows(isFullStop: Boolean = true): SummaryListRow = {
+    cashTransactionsResultRow(CashTransactionDates(from, to), isFullStop, isDay)
+  }
+
+  def rowsV2(isFullStop: Boolean = true): SummaryListRow = {
+    cashTransactionsResultRowV2(CashTransactionDates(from, to), isFullStop, isDay)
   }
 
   private def cashTransactionsResultRow(dates: CashTransactionDates,
-                                        fullStop: Boolean,
-                                        day: Boolean): SummaryListRow = {
+                                        isFullStop: Boolean,
+                                        isDay: Boolean): SummaryListRow = {
     summaryListRow(
-      value = HtmlFormat.escape(
-        if (fullStop) {
-          rowResult(dates, day)
-        } else {
-          rowResultWithoutFullStop(dates, day)
-        }).toString(),
+      value = populateSummaryListRowValue(dates, isFullStop, isDay),
       secondValue = None,
       actions = Actions(items = Seq(ActionItem(
         href = controllers.routes.DownloadCsvController.downloadRequestedCsv(
           None, RequestedDateRange(dates.start, dates.end)).url,
         content = span(messages("cf.cash-account.detail.csv")),
         visuallyHiddenText = Some(messages("cf.cash-account.detail.csv-definition"))
       )))
+    )
+  }
 
+  private def cashTransactionsResultRowV2(dates: CashTransactionDates,
+                                          isFullStop: Boolean,
+                                          isDay: Boolean): SummaryListRow = {
+    summaryListRow(
+      value = populateSummaryListRowValue(dates, isFullStop, isDay),
+      secondValue = None,
+      actions = Actions(
+        items = Seq(ActionItem(href = controllers.routes.SelectTransactionsController.onPageLoad().url))
+      )
     )
   }
 
-  private def rowResult(dates: CashTransactionDates, day: Boolean): String = {
+  private def populateSummaryListRowValue(dates: CashTransactionDates, isFullStop: Boolean, isDay: Boolean): String = {
```

Comment:
Better to keep the parameters on separate line to make it more readable as right now due to method long name and one of the parameter types is also long so not easily readable